collector_k8s/templates/collector.yaml

```diff
@@ -28,14 +39,16 @@ spec:
       {{ if $collector.scrape_configs_file }}
       prometheus:
         config:
-          scrape_configs: {{ $.Files.Get $collector.scrape_configs_file | nindent 10 }}
+          scrape_configs: {{ tpl ($.Files.Get $collector.scrape_configs_file) (dict "collectorName" $collectorName "Template" $.Template) | nindent 10 }}
```

Comment:
This line is a bit wonky because of https://github.com/helm/helm/issues/5979 but it lets us template and bake in what the correct scrape config for the collector will be